### ISSUE TITLE
Metrics dashboard: Yellow doesn't meet WCAG AA

### ISSUE THREAD

**Issue by jmealingcds** [cds-snc/ai-answers#956]

**Comment by jmealingcds** [comment]
https://github.com/cds-snc/ai-answers/pull/954
